project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md

```diff
@@ -0,0 +1,34 @@
+# Sprint 1 Retrospective with AT 
+1st April 2020
+### Attendees
+* Georgia Aitkenhead
+* Thomas Stanley
+* Christippher Legg
+* Emilia Driscoll
+* Ellen Devereux
+* Harrison Cannon
+* Conor Henderson
+* Joe Collins
+* Jake Wych
+* Ryan Gallery
+* Daniel Ball
```

Comment:
Fine for now but I recommend first name and last initial to maintain just a little bit of privacy in these public minutes.

Reply:
Sure